package protoFile.quest.TaskExcuteTalkTaskRequest1419;
//请求执行对话任务

message TaskExcuteTalkTaskRequest{ 
    required int32 id		= 1;			//角色id
	required int32 task_id	= 2;			//任务id
	optional int32 npc_id	= 3;			//npc id
}

message TaskExcuteTalkTaskResponse{
	required bool	result				= 1;			//是否成功
	optional string message				= 2;			//错误说明
}

